### PR TITLE
[W32] Ray Lin: 08/08 - 08/12

### PR DESCRIPTION
**Pull request by raylin**

### CODE REVIEW

File: RayLin.md [hunk 1]
```diff
@@ -1,29 +1,26 @@
...
   - Bug 1271765 - Visual refresh of media controls
-    - Stephen reviewed the concept and mainly agreed
-    - Peko will offer icons in SVG format first and then spec
-  - Bug 1222273 - Convert HTML \<video\>/\<audio\> controls to be (mostly?) HTML instead of XUL
-    - Depends on: 1292083
+    - WIP
+    - :dolske and :jaws gave some feedback.
+    - Got a spec draft from Peko.
+  - Bug 1292083 - Split mobile/desktop video control and make them inherited from a minimal base binding
+    - Deferred
```
Comment (timdream) [inline RayLin.md]: I thought you are going to work on this first? Why the change of plan?

Comment (raylin) [inline RayLin.md]: I was planning to start visual refresh once I got spec and icons. De-XUL process is flexible and doesn't conflict with visual refresh, so I could easily switch back and continue prior work.
